src/components/CytoscapeGraph/CytoscapeGraphUtils.ts

```diff
@@ -108,6 +108,7 @@ export const runLayout = (cy: Cy.Core, layout: Layout): Promise<any> => {
       ...layoutOptions,
       name: 'box-layout',
       appBoxLayout: 'dagre',
+      namespaceBoxLayout: 'dagre',
```

Comment:
We may want to add logic here.  We know  'dagre' fails when there are a lot of orphans (or possibly when not showing edges, based on the stuff I'm looking into).  In that case we may want to set `namespaceBoxLayout: 'grid'`.   For now this is still an improvement.